<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="directory">
        <property name="text">
         <string>Directory isn't selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputString"/>
      </item>
      <item>
       <widget class="QLabel" name="statusString">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listFiles"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIndex"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="actionOpenDirectory"/>
   <addaction name="actionOpenFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionIndex">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icon/index.png</normaloff>:/icon/icon/index.png</iconset>
   </property>
   <property name="text">
    <string>Index</string>
   </property>
  </action>
  <action name="actionOpenDirectory">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icon/openDirectory.png</normaloff>:/icon/icon/openDirectory.png</iconset>
   </property>
   <property name="text">
    <string>Open directory</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icon/openFile.png</normaloff>:/icon/icon/openFile.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icon/run.png</normaloff>:/icon/icon/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icon/srop.png</normaloff>:/icon/icon/srop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
